pkg/kopia/repository/client.go

```diff
@@ -62,10 +62,10 @@ func ConnectToAPIServer(
 
 	opts := &repo.ConnectOptions{
 		CachingOptions: content.CachingOptions{
-			CacheDirectory:            kopia.DefaultClientCacheDirectory,
-			MaxCacheSizeBytes:         int64(contentCacheMB << 20),
-			MaxMetadataCacheSizeBytes: int64(metadataCacheMB << 20),
-			MaxListCacheDuration:      content.DurationSeconds(defaultConnectMaxListCacheDuration.Seconds()),
+			CacheDirectory:              kopia.DefaultClientCacheDirectory,
+			ContentCacheSizeLimitBytes:  int64(contentCacheMB << 20),
```

Comment:
I see `ContentCacheSizeBytes` and `MetadataCacheSizeBytes` in PR by @leuyentran : https://github.com/kanisterio/kanister/pull/2416/files#diff-d96603cf822203f32cb69e1031279e1c357193ab6a01bbcf59ce08caa3dc3f8dR66. Do we know which one of these two should be used?

Reply:
@ankitjain235 In [this](https://github.com/kopia/kopia/pull/3238) Kopia PR for adding hard limits for cache, the soft limit fields in the struct were renamed from `MaxCacheSizeBytes` to `ContentCacheSizeBytes` and `MaxMetadataCacheSizeBytes` to `MetadataCacheSizeBytes`. Since Le's changes are only for kopia upgrade, we are using renamed fields for Soft limits. The actual work of replacing soft limits with hard limits fields would be done in current PR. I wanted to keep the changes of upgrade separate from the changes that are required for hard limit flags. 

So Le's PR should be merged first and then current PR would be merged